lib/IRGen/GenType.h

```diff
@@ -220,6 +220,22 @@ class GenericContextScope {
   }
 };
 
+/// An RAII interface for forcing types to be lowered bypassing resilience.
+class CompletelyFragileScope {
+  TypeConverter &TC;
+public:
+  CompletelyFragileScope(TypeConverter &TC) : TC(TC) {
+    TC.pushCompletelyFragile();
+  }
+
+  CompletelyFragileScope(IRGenModule &IGM)
```

Comment:
Nitpick: can you make these constructors `explicit`?

Reply:
Oops, I usually remember to do that. I'll make a follow up patch.